spatialos-sdk/src/worker/schema.rs

```diff
@@ -0,0 +1,582 @@
+use crate::worker::{component::ComponentId, EntityId};
+use spatialos_sdk_sys::worker::*;
+use std::collections::BTreeMap;
+
+pub type FieldId = u32;
+
+#[derive(Debug)]
+pub struct SchemaComponentUpdate {
+    pub component_id: ComponentId,
+    pub internal: *mut Schema_ComponentUpdate,
```

Comment:
No, this should be private. I'm going to do a pass in a bit to improve encapsulation and see if I can make `SchemaObject` actually safe since it currently has some soundness holes.